### PR TITLE
Refactor allow VPN access

### PR DESCRIPTION
**Pull request by zaro0508**
The VPN setup to allow VPN access using the IpProtocol setting
didn't work.  Refactor to use the CidrIp setting.

### CODE REVIEW

File: templates/mongodb.yaml [hunk 1]
```diff
@@ -288,11 +290,11 @@ Resources:
           FromPort: 28017
           ToPort: 28017
           SourceSecurityGroupId: !Ref MongoDBServerAccessSecurityGroup
-        - IpProtocol: tcp
-          FromPort: 22
-          ToPort: 22
-          SourceSecurityGroupId: !ImportValue
-            'Fn::Sub': '${AWS::Region}-${VpcName}-VpnSecurityGroup'
+        - CidrIp: !Ref VpnCidr
```
Comment (ahayden) [inline templates/mongodb.yaml:293]: This block does not have a `SourceSecurityGroupId` value... this should be the an SG that maps to the VPC that contains the VPN

Comment (zaro0508) [reply templates/mongodb.yaml:293]: I don't understand what you mean @ahayden , could you please restate?

Comment (ahayden) [reply templates/mongodb.yaml:293]: What I mean it, you removed the SourceSecurityGroupId value, and I think it should be there

Comment (zaro0508) [reply templates/mongodb.yaml:293]: hmm, the other blocks that contain `SourceSecurityGroupId` seems to be restricting to a specific port.  Unlike the other blocks this block has no restrictions. all ports are open for VPN.  What would you suggest as the value for SourceSecurityGroupId in this block?

Comment (ahayden) [reply templates/mongodb.yaml:293]: Reuse this import?
```
SourceSecurityGroupId: !ImportValue
            'Fn::Sub': '${AWS::Region}-${VpcName}-VpnSecurityGroup'
```

Comment (zaro0508) [reply templates/mongodb.yaml:293]: I just tried that but got the following error...

```
2020-07-14 20:08:53 UTC-0700 | MongoDBServerSecurityGroup | UPDATE_FAILED | Both CidrIp and SourceSecurityGroup cannot be specified
```

Comment (ahayden) [reply templates/mongodb.yaml:293]: Oh welp, I suppose your initial solution is equivalent.

**Pull request merged**